docs/source/config.json

```diff
@@ -61,11 +61,12 @@
       "Batching operations": "/advanced/query-batching"
     },
     "Development & Testing": {
-      "Overview": "/testing/overview",
+      "Testing overview": "/testing/overview",
       "Mocking HTTP responses": "/testing/mocking-http-responses",
       "Mocking GraphQL responses": "/testing/mocking-graphql-responses",
       "Data builders": "/testing/data-builders",
-      "UI Tests": "/testing/ui-tests"
+      "UI Tests": "/testing/ui-tests",
+      "Android Studio plugin": "/testing/android-studio-plugin"
```

Comment:
I would move it up before testing in the list because you should set this up before you start testing. Also update the "Get Started" to link to this page?